### PR TITLE
Add files for popup task

### PR DESCRIPTION
**Pull request by AnzhelikaKh**
### Popup task

[Demo](https://anzhelikakh.github.io/popup/) |
[Code base](https://github.com/AnzhelikaKh/popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/AnzhelikaKh/popup_task/index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>PopUp</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="logo"><a href="#">
+                <img src="./img/Logo1.jpg" alt="Logo"></a>
+            </div>
```
Comment (A-Ostrovnyy) [inline submissions/AnzhelikaKh/popup_task/index.html:15]: The logo must be a link and lead us to the home page.

File: submissions/AnzhelikaKh/popup_task/index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                    <li class="menu__item">
+                        <input type="checkbox" class="checkbox__main" id="checkbox__main">
+                        <label for="checkbox__main" class="main__label">
+                            <img src="./img/popup-button.png" alt="popup-button">
+                        </label>
+                        <ul class="popup__list">
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img src="./img/apps.png" alt="Apps">
```
Comment (A-Ostrovnyy) [inline submissions/AnzhelikaKh/popup_task/index.html:29]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/AnzhelikaKh/popup_task/index.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <input type="checkbox" class="checkbox__more" id="checkbox__more">
+                                <label for="checkbox__more" class="more__label">
```
Comment (A-Ostrovnyy) [inline submissions/AnzhelikaKh/popup_task/index.html:76]: Logically this button must be hidden when the second part of the popup is open.

File: submissions/AnzhelikaKh/popup_task/style.css [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    height: 1;
+}
+.main__content{
+  text-transform:uppercase;
+  text-shadow: #000;
+  text-align: center;
+  margin: 30px;
+  
+}
+h1{
```
Comment (A-Ostrovnyy) [inline submissions/AnzhelikaKh/popup_task/style.css:160]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

File: submissions/AnzhelikaKh/popup_task/style.css [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+.header{
+    display: flex;
+    align-items: center;
+    height: 90px;
+    border-bottom: 1px solid #000;
+    box-shadow: 0 2px 5px #000;
+
+}
+.logo img{
```
Comment (A-Ostrovnyy) [inline submissions/AnzhelikaKh/popup_task/style.css:22]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css). Also, avoid too specific CSS selectors. [Read about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Comment by A-Ostrovnyy** [comment]
Don't forget about mobile version
image

File: submissions/AnzhelikaKh/popup_task/index.html (hunk 2 above)
Comment (A-Ostrovnyy) [reply submissions/AnzhelikaKh/popup_task/index.html]: For decorative images `alt` must be empty, the icon is a decorative image.

File: submissions/AnzhelikaKh/popup_task/index.html (hunk 3 above)
Comment (A-Ostrovnyy) [reply submissions/AnzhelikaKh/popup_task/index.html:76]: Still not fixed :cry:

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

File: submissions/AnzhelikaKh/popup_task/index.html (hunk 3 above)
Comment (AnzhelikaKh) [reply submissions/AnzhelikaKh/popup_task/index.html:76]: Is the option when the "More" button is hidden and changed to the "Less" button not suitable? 😢

Comment (A-Ostrovnyy) [reply submissions/AnzhelikaKh/popup_task/index.html:76]: Holding the 'More/less' button in the middle of the list doesn't look good. Usually, it keeps after list.

File: submissions/AnzhelikaKh/popup_task/index.html [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>PopUp</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="logo"><a href="./index.html">
+                <img class = "logo__img" src="./img/Logo1.jpg" alt="Logo"></a>
+            </div>
```
Comment (A-Ostrovnyy) [inline submissions/AnzhelikaKh/popup_task/index.html:15]: A logo should be a link and leads users to the home page

File: submissions/AnzhelikaKh/popup_task/index.html [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+                    <li class="menu__item">
+                        <input type="checkbox" class="checkbox__main" id="checkbox__main">
+                        <label for="checkbox__main" class="main__label">
+                            <img src="./img/popup-button.png" alt="popup-button">
+                        </label>
+                        <ul class="popup__list">
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img class="item__view__img" src="./img/apps.png" alt="Icon Apps item">
```
Comment (A-Ostrovnyy) [inline submissions/AnzhelikaKh/popup_task/index.html:29]: For decorative images implemented with `img` tag, `alt` attribute should be an empty value. The icon is a decorative image.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/AnzhelikaKh/popup_task/index.html (hunk 6 above)
Comment (AnzhelikaKh) [reply submissions/AnzhelikaKh/popup_task/index.html:15]: I already have this link "<a href="./index.html">" :)

File: submissions/AnzhelikaKh/popup_task/index.html (hunk 3 above)
Comment (AnzhelikaKh) [reply submissions/AnzhelikaKh/popup_task/index.html:76]: Okey, uno momento! ^^

File: submissions/AnzhelikaKh/popup_task/index.html (hunk 7 above)
Comment (AnzhelikaKh) [reply submissions/AnzhelikaKh/popup_task/index.html:29]: I'm sorry, I already fixed it 👍

File: submissions/AnzhelikaKh/popup_task/index.html [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <input type="checkbox" class="checkbox__more" id="checkbox__more">
+                                <label for="checkbox__more" class="more__label">
+                                    <span class="btn visible">More</span>
+                                </label>
+
+                                <ul class="more__list">
```
Comment (A-Ostrovnyy) [inline submissions/AnzhelikaKh/popup_task/index.html:80]: Houston we have a problem 
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
BTW don't resolve comments in the future, comments must be resolved by the author.)

**Comment by AnzhelikaKh** [comment]
> Good work! BTW don't resolve comments in the future, comments must be resolved by the author.)

Sorry, I will not do that again :)

File: submissions/AnzhelikaKh/popup_task/index.html (hunk 8 above)
Comment (AnzhelikaKh) [reply submissions/AnzhelikaKh/popup_task/index.html:80]: Look like the problem is solved😅

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**